app/views/users/index.html.erb

```diff
@@ -1 +1,6 @@
-<h1>This is a list of users</h1>
+<div class="container">
+  <h1 class="heading">RailsBlog</h1>
+  <% @users.each do |user| %>
+    <%= render partial: "users/user", locals: { user: user } %>
+  <% end %>
```

Comment:
- [ ] [OPTIONAL] Well done for creating `app/views/users/_user.html.erb` partial 👏 👏 However, I have a slight improvement here for you to make. Instead of using `@users.each`, you can just do the below 👇 
```ruby
  <%= render @users %>
```